packages/edit-site/src/components/code-editor/style.scss

```
@@ -1,7 +1,6 @@
 .edit-site-code-editor {
 	position: relative;
 	width: 100%;
-	min-height: 100%;
 	background-color: $white;
```

```suggestion
	flex-glow: 1;
	background-color: $white;
```

Without `flex-grow`, the background color would not stretch to its full height when there is little content.

---

I can confirm this proposal using `flex-grow: 1` fixes the issue and also allow the background-color to cover the entire space when there is less content.